src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -88,23 +89,28 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
             return;
         }
 
-        const initialPage = targetAttachments.findIndex(compareImage);
+        let attachmentIndex = targetAttachments.findIndex(compareImage);
+        const prevAttachmentIndex = attachments.findIndex(compareImage);
```

Comment:
Can you do same refactor on the names on this file too

Reply:
done